### PR TITLE
Fix issue #1824 (If user decided to not modify the .bashrc, we should not show the source .bashrc at the end)

### LINKED ISSUE #1824: If user decided to not modify the .bashrc, we should not show the source .bashrc at the end
Because user selected "n" (see below), the following message should not be displayed at the end of the installation:
~~~
Open a new Terminal window to load environment variables, or run:
 source /Users/julien/.bashrc" 
~~~

It should be replaced by something like:
~~~
To use SCT now, please update your environment variable by running:
export PATH=/Users/julien/code/sct/bin:$PATH
source /Users/julien/code/sct/bin/sct_env
~~~

Below the full verbose (towards the end of the install):
~~~
[...]
Do you want to add the sct_* scripts to your PATH environment? [y]es/[n]o: n

Not adding /Users/julien/code/sct-cnn to $PATH
You can always add it later or call SCT functions with full path /Users/julien/code/sct-cnn/bin/sct_function

Validate installation...

--
Spinal Cord Toolbox (master/f4a3ad5e44e07a37cfd5a0bad1065ada4cbb8632)
Running /Users/julien/code/sct-cnn/scripts/sct_check_dependencies.py
SCT info:
- version: git-master-f4a3ad5e44e07a37cfd5a0bad1065ada4cbb8632
- path: /Users/julien/code/sct-cnn
OS: osx (Darwin-17.4.0-x86_64-i386-64bit)
CPU cores: Available: 8, Used by SCT: 8
RAM: Primary memory available: 16.00 gigabytes
Check Python executable.............................[OK]
  Using bundled python 2.7.12 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:43:17) 
[GCC 4.2.1 (Based on Apple Inc. build 5658) (LLVM build 2336.11.00)] at /Users/julien/code/sct-cnn/python/bin/python
Check if data are installed.........................[OK]
Check if xlwt (1.0.0) is installed..................[OK]
Check if xlutils (1.7.1) is installed...............[OK]
Check if cryptography (1.6) is installed............[OK]
Check if scikit-learn (0.17.1) is installed.........[OK]
Check if scikit-image (0.12.3) is installed.........[OK]
Check if pyqt (4.11.4) is installed.................[OK]
Check if psutil (5.2.2) is installed................[OK]
Check if matplotlib (1.5.1) is installed............[OK]
Check if pip (9.0.1) is installed...................[OK]
Check if requests (2.12.4) is installed.............[OK]
Check if xlrd (0.9.4) is installed..................[OK]
Check if pandas (0.18.1) is installed...............[OK]
Check if dipy (0.11.0) is installed.................[OK]
Check if tqdm (4.11.2) is installed.................[OK]
Check if nibabel (2.1.0) is installed...............[OK]
Check if Keras (2.0.6) is installed.................[OK]
Check if futures (3.2.0) is installed...............[OK]
Check if pytest-cov (2.5.1) is installed............[OK]
Check if pytest (3.5.0) is installed................[OK]
Check if nipy (0.4.1) is installed..................[OK]
Check if h5py (2.7.0) is installed..................[OK]
Check if tensorflow (1.3.0) is installed............[OK]
Check if raven (6.2.1) is installed.................[OK]
Check if numpy is installed (1.11.2)................[OK]
Check if scipy is installed (0.18.1)................[OK]
Check if spinalcordtoolbox is installed.............[OK]
Check ANTs compatibility with OS ...................[OK]
Check PropSeg compatibility with OS ................[OK]
Check if figure can be opened.......................[OK]

Installation validation OK!

Open a new Terminal window to load environment variables, or run:
 source /Users/julien/.bashrc
~~~

### PR DESCRIPTION
**Pull request by abtahizadeh**
### Requirements

* [* ] Is there a issue open describing the motivation for this pull request ?
#1824 
* [ ] Does the pull request need an update of the documentation ?
* [*] Does the pull request need additional test(s) ?
Installation test required
* [ ] Clean up the warnings generated by the [style guide](https://www.jetbrains.com/help/pycharm/2017.1/code-inspection.html)

### Description of the Change
Adding a condition to check the user input and making it properly verbose at the end of the installation.

### Steps and Constraints

[The steps of reproducing the pull request]

### Applicable Issues

Implements or Fixes #1824

### CODE REVIEW

**Comment by abtahizadeh** [comment]
Ignore previous commits (already merged)

File: install_sct [hunk 1]
```diff
@@ -616,7 +616,13 @@ if [[ $UID != 0 ]]; then
   cd ${TMP_DIR}
   if ${SCT_DIR}/${BIN_DIR}/sct_check_dependencies; then
```
Comment (abtahizadeh) [inline install_sct]: Possible to make it one-liner with "&&" as well.

File: install_sct [hunk 2]
```diff
@@ -616,7 +616,13 @@ if [[ $UID != 0 ]]; then
...
     echo -e "Installation validation OK!\n"
-    echo -e "Open a new Terminal window to load environment variables, or run:\n source ${THE_RC}\n"
+    if [[ ${add_to_path} =~ ^[Nn] ]]; then
+      echo -e "To use SCT now, please update your environment variable by running:"
+      echo -e "export PATH=${SCT_DIR}/${BIN_DIR}:$PATH\n"
+      echo -e "source ${SCT_DIR}/${BIN_DIR}/sct_env\n"
+    else
+      echo -e "Open a new Terminal window to load environment variables, or run:\n source ${THE_RC}\n"
```
Comment (jcohenadad) [inline install_sct]: "\n" not compatible with echo on OSX

Comment (jcohenadad) [reply install_sct]: my bad-- "-e" fixes it :-)

**Comment by zougloub** [comment]
Please fix the history of your git branch by rebasing it on master and discarding noop commits if needed.

**Pull request closed without merging**